app/controllers/static_pages_controller.rb

```diff
@@ -0,0 +1,11 @@
+class StaticPagesController < ApplicationController
+
+  def home
```

Comment:
```suggestion
  def home; end
```
tuong tu method khac